### DISCUSSION TITLE
Libmesh compilation failed on HPC

### DISCUSSION THREAD

**Discussion by aashiquear** [idaholab/moose#21727] [Q&A General] [answered]
I am using the following to setup MOOSE on HPC,

gcc 9.3.1
python 3.6.5
cmake 3.22.3
openmpi-gnu 3.1

After PETSc compilation I am getting the following error during libmesh compilation,

`../../tests/mesh/mesh_triangulation.C: In member function ‘void MeshTriangulationTest::testTriangulatorBase(libMesh::MeshBase&, libMesh::TriangulatorInterface&)’:
../../tests/mesh/mesh_triangulation.C:148:9: error: ‘CPPUNIT_ASSERT_GREATER’ was not declared in this scope; did you mean ‘CPPUNIT_ASSERT_EQUAL’?
  148 |         CPPUNIT_ASSERT_GREATER(Real(0), cross_prod(2));
      |         ^~~~~~~~~~~~~~~~~~~~~~
      |         CPPUNIT_ASSERT_EQUAL`

How can I resolve the setup?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Did you update recently?
Seems you have an old CPPUNIT with a recent libmesh

Guillaume
